src/core/system.js

```diff
@@ -35,13 +38,9 @@ var System = module.exports.System = function (sceneEl) {
   if (component) { component.Component.prototype.system = this; }
 
   // Process system configuration.
-  if (!Object.keys(schema).length) { return; }
-  rawData = HTMLElement.prototype.getAttribute.call(sceneEl, this.name);
-  if (isSingleProp(schema)) {
-    this.data = parseProperty(rawData, schema);
-    return;
-  }
-  this.data = parseProperties(styleParser.parse(rawData) || {}, schema, false, this.name);
+  this.buildData();
```

Comment:
`updateProperties` doesn't call `init()`, and `init()` needs the data when it's run. So need to run them separately.